src/apify_client/consts.py

```diff
@@ -1,87 +1,29 @@
-        """Whether this actor job status is terminal."""
-        return self in (ActorJobStatus.SUCCEEDED, ActorJobStatus.FAILED, ActorJobStatus.TIMED_OUT, ActorJobStatus.ABORTED)
-
-
-class ActorSourceType(str, Enum):
-    """Available source types for actors."""
-
-    #: Actor source code is comprised of multiple files
-    SOURCE_FILES = 'SOURCE_FILES'
-    #: Actor source code is cloned from a Git repository
-    GIT_REPO = 'GIT_REPO'
-    #: Actor source code is downloaded using a tarball or Zip file
-    TARBALL = 'TARBALL'
-    #: Actor source code is taken from a GitHub Gist
-    GITHUB_GIST = 'GITHUB_GIST'
-
-
-class WebhookEventType(str, Enum):
-    """Events that can trigger a webhook."""
-
-    #: The actor run was created
-    ACTOR_RUN_CREATED = 'ACTOR.RUN.CREATED'
-    #: The actor run has succeeded
-    ACTOR_RUN_SUCCEEDED = 'ACTOR.RUN.SUCCEEDED'
-    #: The actor run has failed
-    ACTOR_RUN_FAILED = 'ACTOR.RUN.FAILED'
-    #: The actor run has timed out
-    ACTOR_RUN_TIMED_OUT = 'ACTOR.RUN.TIMED_OUT'
-    #: The actor run was aborted
-    ACTOR_RUN_ABORTED = 'ACTOR.RUN.ABORTED'
-    #: The actor run was resurrected
-    ACTOR_RUN_RESURRECTED = 'ACTOR.RUN.RESURRECTED'
-
-    #: The actor build was created
-    ACTOR_BUILD_CREATED = 'ACTOR.BUILD.CREATED'
-    #: The actor build has succeeded
-    ACTOR_BUILD_SUCCEEDED = 'ACTOR.BUILD.SUCCEEDED'
-    #: The actor build has failed
-    ACTOR_BUILD_FAILED = 'ACTOR.BUILD.FAILED'
-    #: The actor build has timed out
-    ACTOR_BUILD_TIMED_OUT = 'ACTOR.BUILD.TIMED_OUT'
-    #: The actor build was aborted
-    ACTOR_BUILD_ABORTED = 'ACTOR.BUILD.ABORTED'
-
-
-class MetaOrigin(str, Enum):
-    """Possible origins for actor runs, i.e. how were the jobs started."""
-
-    #: Job started from Developer console in Source section of actor
-    DEVELOPMENT = 'DEVELOPMENT'
-    #: Job started from other place on the website (either console or task detail page)
-    WEB = 'WEB'
-    #: Job started through API
-    API = 'API'
-    #: Job started through Scheduler
-    SCHEDULER = 'SCHEDULER'
-    #: Job started through test actor page
-    TEST = 'TEST'
-    #: Job started by the webhook
-    WEBHOOK = 'WEBHOOK'
-    #: Job started by another actor run
-    ACTOR = 'ACTOR'
+import warnings
+
+from apify_shared.consts import (
+    BOOL_ENV_VARS,
+    BOOL_ENV_VARS_TYPE,
+    DATETIME_ENV_VARS,
+    DATETIME_ENV_VARS_TYPE,
+    FLOAT_ENV_VARS,
+    FLOAT_ENV_VARS_TYPE,
+    INTEGER_ENV_VARS,
+    INTEGER_ENV_VARS_TYPE,
+    STRING_ENV_VARS,
+    STRING_ENV_VARS_TYPE,
+    ActorEventTypes,
+    ActorExitCodes,
+    ActorJobStatus,
+    ActorSourceType,
+    ApifyEnvVars,
+    MetaOrigin,
+    WebhookEventType,
+)
+
+# Trigger DeprecationWarning
+warnings.warn(
+    ('Importing constants from "apify_client.consts" is deprecated and will be removed in the future. '
+     'Please use "apify_shared" library instead.'),
+    category=DeprecationWarning,
+    stacklevel=2,
+)
```

Comment:
I think we can't just put a warning when importing this module, we might at some point want to add some non-deprecated client-only constants here (and we do have SDK-only constants in the SDK).

We could solve this by overriding the `__getattr__()` method in this module, as per [PEP 562](https://peps.python.org/pep-0562/).

Reply:
You're right, updated, thanks.